Logger/FileLogger.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Logger;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        public FileLogger(string path)
+        {
+            FilePath = path;
+            ClassName = nameof(FileLogger);
+        }
+        private string FilePath { get; set; }
+
+        public override void Log(LogLevel level, string message)
+        {
+            DateTime date = DateTime.Now;
+            string outLog = $"{date} {ClassName} {Enum.GetName(typeof(LogLevel), level)}: {message}";
```

Comment:
```suggestion
            string outLog = $"{date} {ClassName} {level}: {message}";
```
Using your LogLevel in string interpolation like this will retrieve the name of the enum value for you.